bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html

```diff
@@ -452,9 +452,15 @@ <h3 class="topic-action-title">{{ _('What Mozilla is doing about government surv
       <h2>{{ _('Hungry for more?') }}</h2>
       <p class="report-cta">
         <span>
-        {% trans healthreport='https://internethealthreport.org' %}
-          Read on about Digital Inclusion in our <a href="{{ healthreport }}">Internet Health Report</a>.
-        {% endtrans %}
+        {% if l10n_has_tag('internet_health_subpage_rebrand') %}
+          {% trans healthreport='https://internethealthreport.org' %}
+            Read on about Digital Inclusion in our <a href="{{ healthreport }}">Internet Health Report</a>.
```

Comment:
Looking at the commit history I would hazard a guess that this string should not have been changed (it looks like a `<span>` was added for decoration, but the text might have been a copy/pasta). Will defer to Craig, but i would assume that the fix should be to just revert the string change.